Update V1.5.3

Not-Bay opened the pull request:
Mostly hotfixes

* Fix discord bot commands (make sure you enable message content intent in [discord developer portal](https://discord.com/developers/applications))
* Fix missing requirements on PC (tested with 3.9.16)
* Update `crowns` command, it will do the eid_coronet emote every time you change crowns (and now it checks you are actually entering numbers to it)
* some language files fixes

BayGamerYT commented:
Already in `dev` branch https://github.com/BayGamerYT/Fortnite-LobbyBot-v2/commit/059d70accff48b7e304e6518abd688611ab6870f

Not-Bay committed:
Update fortnitepy source

There is no `gomashio1596/fortnitepy` anymore

Not-Bay committed:
Create .gitignore

Not-Bay committed:
Create replit.nix

Not-Bay committed:
Set nix channel to stable 22_11

Not-Bay committed:
Update requirements for PC

Not-Bay committed:
Fix discord bot commands

Updates discord.py to latest version (2.2.2) and configures intents for the client
Message Content Intent must be enabled in developer portal

Not-Bay committed:
Update `crowns` command

- Show usage if input is not a number
- Do eid_coronet emote every time the command is used

Not-Bay committed:
Add missing keys

Not-Bay committed:
some corrections/adjustments

Should be more understandable now

Not-Bay committed:
version bump
